decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb

```diff
@@ -109,7 +109,8 @@ def meeting
       end
 
       def meetings
-        @meetings ||= paginate(search.result.order(start_time: :desc))
+        is_past_meetings = params.dig("filter", "with_any_date")&.include?("past") || @forced_past_meetings
```

Comment:
I think you don't need the `@forced_past_meetings` because if that is true, it already overrides the value in the params:
```suggestion
        is_past_meetings = params.dig("filter", "with_any_date")&.include?("past")
```

See:
https://github.com/decidim/decidim/blob/e0370c273a42d40f92d82b57ee5a8922ed53d0bc/decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb#L51

On that line, can you also change the array to a string the same way as in the default filter params?